bolt.gemspec

```diff
@@ -48,7 +48,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "net-ssh", ">= 4.0"
   spec.add_dependency "net-ssh-krb", "~> 0.5"
   spec.add_dependency "orchestrator_client", "~> 0.4"
-  spec.add_dependency "puppet", [">= 6.16.0", "< 7"]
+  spec.add_dependency "puppet", "6.17.0"
```

Comment:
This will break vanagon packaging https://github.com/puppetlabs/puppet-runtime/blob/master/configs/components/rubygem-puppet.rb

Reply:
Updated